src/sentry/api/endpoints/sentry_app_requests.py

```diff
@@ -32,12 +35,25 @@ def format_request(self, request, sentry_app):
         return formatted_request
 
     def get(self, request, sentry_app):
+        """
+        :qparam string eventType: Optionally specify a specific event type to filter requests
+        :qparam bool errorsOnly: If this is true, only return error/warning requests (300-599)
+        """
 
-        # TODO add optional query params for event type
-        # for now I'm just getting all requests for all events
+        event_type = request.GET.get("eventType")
+        errors_only = request.GET.get("errorsOnly")
+
+        kwargs = {}
+        # if somehow the event type is invalid, just ignore it and return all events
+        if event_type and event_type in EXTENDED_VALID_EVENTS:
```

Comment:
@nolachen what happens if `event_type` is an invalid event? I would think we should throw a 400 error.